src/docs/operator-guides/how-to-edit-pages/index.md

```diff
@@ -0,0 +1,295 @@
+
+You can add the title and description you want your users to see. The
+title will be displayed in bigger font at the top of the section. How
+big is determined by your title header level. H1 is bigger than H2. The
+title size also determines the HTML level header indexed by search
+engines. You can use the title to explain the content of the section or
+define a larger text that you want to highlight.
+
+The description refers to the subtitle of the section. It can be used as
+a short or long description for the section. These texts are optional.
+
+![Carousel image with title and description example](/title-and-description.png 'Title and description example')
+
+### Add call to action content (Optional)
+
+The call-to-action content is basically a button in your Section. The
+button can help you guide your users to specific actions that you want
+them to take. That is why it is named call to action. You can define the
+type of call to action, the text and the specific link.
+
+- Call to action type: No call to action, internal link or external
+  link. This determines the default behaviour and validation for the
+  button.
+  - **No call to action** means that there will be no general button for
+    this Section. You can always add buttons via call to action settings
+    withing blocks
+  - **Internal link** means that the link will open in the same tab. The
+    link should point to an internal folder within your marketplace,
+    with a partial URL.
+  - **External link** means that the link will open in a new tab. You
+    need to specify the complete URL.
+- Link text: The text within the call to action button. Try to make the
+  text short and simple to engage your visitors and incite them to
+  action.
+- Link address: The URL or folder for the button.
+  - **Internal link:** you have to specify a folder within your
+    marketplace. For example “/p/about”, it will take your users to
+    `https://yourmarketplaceurl.com/p/about`
+  - **External link:** you have to specify a complete URL. You need to
+    add http:// or https:// to your link.
+
+Adding a call to action to your Section is optional.
+
+### Specify Section appearance
+
+You can choose between Default or Custom.
+
+**Default**
+
+The default background is white or shaded white. The color alternates
+within each Section to make a slight yet almost imperceptible difference
+between them. Between rgb(252,252,252) and rgb(250,250,250)
+
+**Custom**
+
+With custom appearance, you can specify any other background color with
+the color picker or add a background image to a section. The image will
+always be set on top of the background color. Unless your image is
+transparent, it will cover the background color. Your image will be
+expanded to fit the dimensions of the section. You can also define the
+text color. The options are black and white. The background and the
+theme affect the whole section, including all the blocks contained in
+it.
+
+## Add blocks (Optional)
```

Comment:
I would say "Content Blocks" in this title to reinforce the idea that this is where your content should mainly go